h3i/src/client/sync_client.rs

```diff
@@ -66,13 +66,6 @@ pub fn connect(
     let mut poll = mio::Poll::new().unwrap();
     let mut events = mio::Events::with_capacity(1024);
 
-    // We'll only connect to one server.
-    let connect_url = if !args.omit_sni {
-        args.host_port.split(':').next()
-    } else {
-        None
-    };
-
     // Resolve server address.
     let peer_addr = if let Some(addr) = &args.connect_to {
```

Comment:
This code is now duplicated in `build_quiche_connection()`, so we should find a way to avoid that

Reply:
Ah missed that one, thanks